udata_gouvfr/theme/gouvfr/templates/header.html

```diff
@@ -1,11 +1,20 @@
+<div class="bg-blue-350">
+    <div class="hagrid banner">
+        <div class="row align-items-center text-align-center-sm">
+            <p class="text-white">
+                Bienvenue sur la nouvelle version <i>beta</i> de data.gouv.fr&nbsp;! <a href="">Apprenez-en plus</a> ou revenez <a href="https://legacy.data.gouv.fr">à l'ancienne version.</a>
```

Comment:
We should write it in English and add the French translation I guess.

Reply:
Not sure this is worth it for such a temporary thing.

Reply:
Depends if we communicate on the beta at an international level ?

Reply:
True. I'll change it.